root/components/relationships-table.tt

```diff
@@ -31,10 +31,17 @@
                     [%~ END ~%]
                     <td>
                       [%~ IF rel.link.attributes.size ~%]
-                      [%~ attr_list = [];
+                      [%~ attr_list1 = []; attr_list2 = [];
+                          instrument_root = '0abd7f04-5e28-425b-956f-94789d9bcbe2';
+                          vocals_root     = 'd92884b7-ee0c-46d5-96f3-918196ba8c5b';
```

Comment:
I saw the IRC discussion about this, but since we have `$INSTRUMENT_ROOT_ID` and `$VOCAL_ROOT_ID` constants already that are used all over the place, we should use them here, too. This is internal to the server; stability of the MBIDs doesn’t matter here.